downstream/modules/eda/ref-eda-logging-samples.adoc

```diff
@@ -0,0 +1,77 @@
+[id="eda-logging-samples"]
+
+= Logging samples
```

Comment:
On second thought, lets remove all sample logs here. This is likely just going to clutter the docs, and we already note above that we support simple logging on all supported operations of supported resources.